src/RoslynInsertionTool/RoslynInsertionTool.Commandline/RoslynInsertionToolCommandline.ShowHelp.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
```

Comment:
Missing copyright

Reply:
Sigh. I guess somehow we never ended up with an analyzer for copyright headers?

Reply:
@dpoeschl